src/ucs/stats/stats.c

```diff
@@ -547,8 +622,12 @@ static void ucs_stats_clean_node_recurs(ucs_stats_node_t *node)
         ucs_warn("stats node "UCS_STATS_NODE_FMT" still has active children",
                  UCS_STATS_NODE_ARG(node));
     }
-
+//int end_less = 0;
     ucs_list_for_each_safe(child, tmp, &node->children[UCS_STATS_INACTIVE_CHILDREN], list) {
+//if (end_less++ > 20) {
```

Comment:
sorry, this commit is not for merge. I was debugging something that reconstruct only on CI